Resources/public/templates/tabs/relations.hbt

```diff
@@ -22,7 +22,17 @@
             {{#each relatedContents}}
             <tr>
                 <td>
-                    <a href="{{path "viewLocation" id=mainLocationId languageCode=content.mainLanguageCode}}">{{content.name}}</a>
+                    <ul class="ez-breadcrumbs-list">
+                        {{#each location.path}}
+                        <li class="ez-breadcrumbs-item">
```

Comment:
That's how indention is done when mixing HTML with handlebars template functions. What's more, HTML generated with that approach will be well-indented. But of course I can change it if it's against used CS. ping @dpobel